### ISSUE TITLE
cron is re-enabled on da1, figure out how to auto disable it

### ISSUE THREAD

**Issue by RyanSquared** [hashbang/shell-server#54]
cron bumped up load from at least 50 to 200 with a massive amount of processes

**Comment by daurnimator** [comment]
Manually disabled and masked on ny1.

Lets mask cron and atd permanently.

**Comment by RyanSquared** [comment]
```
systemd:
  name: "{{ item }}"
  masked: true
with_items:
- cron
- atd
```

**Comment by KellerFuchs** [comment]
I fixed manually at first, #69 has a proper fix.
